packages/sitecore-jss-angular/package.json

```diff
@@ -24,46 +24,50 @@
     "url": "https://github.com/sitecore/jss/issues"
   },
   "devDependencies": {
-    "@angular-eslint/eslint-plugin": "2.0.2",
-    "@angular/common": "~11.2.6",
-    "@angular/compiler": "~11.2.6",
-    "@angular/compiler-cli": "~11.2.6",
-    "@angular/core": "~11.2.6",
-    "@angular/platform-browser": "~11.2.6",
-    "@angular/platform-browser-dynamic": "~11.2.6",
-    "@angular/router": "~11.2.6",
+    "@angular-devkit/build-angular": "^14.2.0",
+    "@angular-eslint/eslint-plugin": "^14.4.0",
+    "@angular/cli": "~14.2.0",
+    "@angular/common": "~14.2.0",
+    "@angular/compiler": "~14.2.0",
+    "@angular/compiler-cli": "~14.2.0",
+    "@angular/core": "~14.2.0",
+    "@angular/platform-browser": "~14.2.0",
+    "@angular/platform-browser-dynamic": "~14.2.0",
+    "@angular/router": "~14.2.0",
     "@types/jasmine": "^3.4.1",
     "@types/node": "^14.14.35",
     "codelyzer": "^6.0.1",
     "eslint": "^7.22.0",
     "jasmine-core": "^3.5.0",
     "karma": "^6.3.2",
     "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage": "~2.2.0",
     "karma-edge-launcher": "^0.4.2",
     "karma-firefox-launcher": "^1.2.0",
-    "karma-jasmine": "^2.0.1",
-    "karma-typescript": "^5.0.3",
-    "ng-packagr": "^11.2.4",
+    "karma-jasmine": "~5.1.0",
+    "karma-jasmine-html-reporter": "~2.0.0",
```

Comment:
`npm i` is not working, cause `karma-jasmin-html-reporter` requires to use `jasmin-core@^4.0.0`

Reply:
npm i worked for me - but I've bumped the jasmine-core version